unittest/app/app.js

```diff
@@ -249,6 +249,11 @@ describe('/tests/', () => {
     assert.equal(res.status, 200);
   });
 
+  it('get all tests, ignore CSS', async () => {
+    const res = await agent.get('/tests/?ignore=css');
+    assert.equal(res.status, 200);
```

Comment:
Should something be asserted here that's different from the previous step? As it is, this test would pass even if the `ignore` parameter is unused.

Reply:
We should probably also check to make sure we're loading the correct tests as well in the previous two steps.  I'm not sure how to do it at this time though, so I've added some TODO comments to remind us later.